funding/manager.go

```diff
@@ -3459,6 +3670,17 @@ func (f *Manager) handleFundingLocked(peer lnpeer.Peer,
 		return
 	}
 
+	// If this is a taproot channel, then we can generate the set of nonces
+	// the remote party needs to send the next remote commitment here.
+	var firstVerNonce *musig2.Nonces
```

Comment:
think we need to do a similar thing as `sendFundingLocked` does, first check the map `f.pendingMusigNonces` then only create one if it doesn't exist, as we don't know which comes first - whether we send one or we receive one.

Reply:
can you elaborate? don't think anything needs to be done here

Reply:
This just pregenerates it, since we'll need it earlier if this is an scid channel upgrade.